src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/PrisonAppointment.kt

```diff
@@ -39,18 +38,8 @@ class PrisonAppointment private constructor(
   val startTime: LocalTime,
 
   val endTime: LocalTime,
-
-  val createdBy: String,
-
-  val createdTime: LocalDateTime = LocalDateTime.now(),
```

Comment:
Have we lost these because they always reflect the same values as the booking itself?

Reply:
Yes